js/react_native/package.json

```diff
@@ -1,6 +1,9 @@
 {
   "react-native": "lib/index",
   "module": "dist/module/index",
+  "engines": {
+    "node": ">=16"
```

Comment:
Why is this change needed? NodeJS 16 is already EOL for many years. Did you hit any issue without this change?
And why is it 16, not 18 or 20?

Reply:
It change was first introduced in [0.71](https://react-native-community.github.io/upgrade-helper/?from=0.70.15&to=0.71.19&package=onnxruntime-android&name=onnxruntime). I guess that the time NODE 16 wasn't EOL. But the current our CI use is >= 16, we should be good. Also for next version upgrade (0.72 to 0.73) this line will be changed to `"node": ">=16"`